### ISSUE TITLE
base64 skips first argument and waits for stdin

### ISSUE THREAD

**Issue by binjip978** [u-root/u-root#2498]
**Describe the bug**
base64 waits for stdin when one parameter was passed. If two parameters were passed it encodes first argument.

**To Reproduce**
Steps to reproduce the behaviour:
```sh
./base64 file.txt
# waits for stdin
./base64 file.txt file.txt
YmFzZTY0LWJ1%
```

**Expected behavior**
Should encode with one argument, like GNU or macOS base64.

**Additional context**
Probably happens because flag.Args return length of non command arguments, with one parameter it length is equal 1 and base64 waits for stdin. The change of > to >= should fix the problem.

**Comment by rminnich** [comment]
Thanks, I'm looking at this now.

**Comment by rminnich** [comment]
fixed by #2499

**Comment by 10000TB** [comment]
@binjip978  fix is in, try from HEAD, and let us know !
